wmn-data.json

```diff
@@ -4868,6 +4868,14 @@
         "known" : ["john", "bob"],
         "cat" : "images"
        },
+       {
+        "name" : "SimplePlanes",
+        "uri_check" : "https://www.simpleplanes.com/u/{account}",
+        "e_code" : 200,
+        "m_code" : 302,

```

Comment:
Let's add an `e_string` of `<h5>joined`

Reply:
Sure

Reply:
Sorry. This is not "resolved" until you commit the e_string/m_string to this PR.